src/System.Collections/src/System/Collections/Generic/SortedSet.TreeSubSet.cs

```diff
@@ -17,6 +17,10 @@ internal sealed class TreeSubSet : SortedSet<T>, ISerializable, IDeserialization
         {
             private SortedSet<T> _underlying;
             private T _min, _max;
+            // keeps track of whether the count variable is up to date
+            // up to date -> countVersion = _underlying.version
+            // not up to date -> countVersion < _underlying.version
+            private int countVersion;
```

Comment:
Could we name this with `_` I think the reason why version doesn't have the prefix is for serialization compatibility, since this is a new field we're adding, we can add the prefix.